db/migrate/20190717030153_add_index_to_users_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsersEmail < ActiveRecord::Migration[5.2]
+  def change
+    add_index :users, :email, unique: true
```

Comment:
@shaniajubelag

Since this migration is in same PR with the **users table creation** (`db/migrate/20190717025920_create_users.rb`),

...it would be better to just add this `email` attribute to the `create_users.rb` migration file